### PR TITLE
Remove some unneeded testing

### PR DESCRIPTION
**Pull request by kripken**
Add a new `@only_wasm2js` decoration for tests that test basic codegen that
makes sense to do only in wasm2js (as for the wasm backend in general we
assume basic codegen is tested in LLVM; we could do the same for wasm2js
which is in Binaryen, but I think some of the Emscripten tests are a good
complement to there).

### CODE REVIEW

**Comment by kripken** [comment]
Also includes a tiny fix to the test runner so that `test_hello_argc` (which replaces hello world) passes in JSC - we need to identify JSC properly in `jsrun`.

File: test/test_core.py [hunk 1]
```diff
@@ -406,11 +414,6 @@ def get_bullet_library(self, use_cmake):
                             configure_args=configure_args,
                             cache_name_extra=configure_commands[0])
 
-  @also_with_standalone_wasm()
-  @also_with_wasmfs
-  def test_hello_world(self):
-    self.do_core_test('test_hello_world.c')
```
Comment (sbc100) [inline test/test_core.py:412]: I kind of like having this tests as the most basic simple test in the suite.  Its the one I normally run before anything else.

**Review by sbc100 — COMMENTED** [review]
lgtm % comment

File: test/test_core.py (hunk 1 above)
Comment (kripken) [reply test/test_core.py:412]: Ok, reverted.

**Pull request merged**